addons/l10n_in/models/account_move_reversal.py

```diff
@@ -0,0 +1,14 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+class AccountMoveReversal(models.TransientModel):
+    _inherit = "account.move.reversal"
+
+    def reverse_moves(self, is_modify=False):
+        res = super().reverse_moves(is_modify)
+        if self.company_id.country_id.code == "IN":
```

Comment:
```suggestion
        if self.env.company.account_fiscal_country_id.code == "IN":
```
It's more robust this way.

Reply:
How is it more robust to use the environment when there is a field on the model? :sweat_smile:

Reply:
Robust (poor choice of word) in the sense that they can be different and we want to use the `account_fiscal_country_id`. For example, if we have a company in a country but mainly doing business in another country.

Reply:
* Have one company in India and one oustside of India
* Have company selector on both companies
* Select moves from both companies
* Duplicate
* :boom:

Reply:
Hello @william-andre ,
So should we go with country_id field or not?
Thank you.